components/PyTorch/pytorch-kfp-components/pytorch_kfp_components/types/standard_component_specs.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env/python3
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
nit: switch to Apache-2.0 license?

Reply:
@Bobgy Thanks for pointing it out. Done.